common/walker/src/report.rs

```diff
@@ -126,6 +129,86 @@ impl SplitScannerError for Result<Report, ScannerError> {
 /// Handle the report
 pub async fn handle_report(report: Report) -> anyhow::Result<()> {
     // FIXME: this is a very simplistic version of handling the error
-    log::info!("Import report: {report:#?}");
+    log::warn!("Import report: {report:#?}");
+
+    let path = env::var_os("REPORT_PATH").unwrap_or_else(|| OsString::from(DEFAULT_REPORT_OUTPUT_PATH));
+    let path = path.to_str().unwrap_or(DEFAULT_REPORT_OUTPUT_PATH).to_string();
+
+    let report_type = env::var_os("REPORT_TYPE").unwrap_or_else(|| OsString::from(DEFAULT_REPORT_TYPE));
+    let report_type = report_type.to_str().unwrap_or(DEFAULT_REPORT_TYPE).to_string();
+
+    let template_content = include_str!("../templates/report.html");
+    let mut tera = Tera::default();
+    tera.add_raw_template("report.html", template_content)?;
+
+    let mut context = Context::new();
+    let current_time = OffsetDateTime::now_utc();
+    context.insert("report", &report);
+    context.insert("type", &report_type);
+    context.insert("current_time", &current_time);
+
+    match tera.render("report.html", &context) {
+        Ok(rendered_html) => {
+            let out_put_path = format!("{path}/report-{current_time:#?}.html");
```

Comment:
Can we format the time so that it's a bit more CLI friendly. Like `report-2024-05-08-09.10.11`